src/charm.py

```diff
@@ -120,6 +120,7 @@ def _pgbouncer_layer(self) -> Layer:
                 PGB: {
                     "summary": "pgbouncer service",
                     "user": PG_USER,
+                    "group": PG_USER,
```

Comment:
LGTM. Nit: I would add constant PG_GROUP to avoid questions and confusion in the future :-)

Reply:
Thank you for addressing it!